src/routes/viewer/PhaseSelect/Combobox/index.tsx

```diff
@@ -2,12 +2,18 @@ import { PhaseSelectProps } from "..";
 import GroupSelect from "./GroupSelect";
 import RankingSelect from "./RankingSelect";
 
-export type PhaseSelectComboboxProps = PhaseSelectProps & {
-  open: [boolean, boolean];
-  setOpen: (group: boolean, ranking: boolean) => void;
+type setOpenType = (group: boolean, ranking: boolean) => void;
+
+type openType = [boolean, boolean];
+
+type comboboxProps = {
+  open: openType;
+  setOpen: setOpenType;
   showGroups: boolean;
 };
 
+export type PhaseSelectComboboxProps = PhaseSelectProps & comboboxProps;
+
```

Comment:
Stesso discorso di prima, non è corretto estrarre types in questoe modo, è accettato tenere definizioni di stato e update stato inline